lib/Commands/BuildEngineCommand.cpp

```diff
@@ -104,7 +104,7 @@ struct AckermannKey {
   /// Convert an Ackermann key to its encoded representation.
   operator core::KeyType() const {
     char inputKey[32];
-    sprintf(inputKey, "ack(%d,%d)", m, n);
+    snprintf(inputKey, sizeof(inputKey), "ack(%d,%d)", m, n);
```

Comment:
@neonichu I'm not sure what you mean, I think `sizeof` is appropriate for this use case. What else would you write?